bazel/BUILD

```diff
@@ -134,6 +134,7 @@ cc_library(
         "-Wno-strict-overflow",
         '-D NGRAPH_BRIDGE_STATIC_LIB_ENABLE',
         '-D NGRAPH_TF_USE_GRAPPLER_OPTIMIZER',
+        '-D NGRAPH_CPU_STATIC_LIB_ENABLE',
```

Comment:
Do we build only CPU for bazel? INTERPRETER backend is not build?

Reply:
We did not need to for the application that use bazel like hello_tf an infer_multi. But if needed in future we could add a rule for INTERPRETER as well